stl/inc/limits

```diff
@@ -1026,7 +1026,7 @@ inline bool _Is_tzcnt_supported() {
 
 
 // see "Hacker's Delight" section 5-3
-template <typename _Ty>
+template <class _Ty>
 constexpr int _Countl_zero_helper(_Ty _Val) {

```

Comment:
It looks like you ran out of `_NODISCARD`s after sprinkling them on the other file - these function templates are all pure observers as well.